### PR TITLE
ConfirmationModal

### PR DESCRIPTION
**Pull request by SMITH1103**
**Description**:
-- Improved appearance of confirmation and logout modals.

### CODE REVIEW

File: src/components/interface/ConfirmationModal.vue [hunk 1]
```diff
@@ -14,17 +14,17 @@
         <PasswordInput show-eye />
```
Comment (jackthta) [inline src/components/interface/ConfirmationModal.vue:14]: Setting the input's placeholder to empty might be good to remove the repetitive prompt 🤔 

Screen Shot 2021-09-20 at 14 01 53

File: src/components/interface/LogoutModal.vue [hunk 2]
```diff
@@ -12,18 +12,18 @@
         </div>
       </div>
 
-      <div class="flex items-center pb-4 mx-4">
+      <div class="mr-4 text-center">
```
Comment (jackthta) [inline src/components/interface/LogoutModal.vue:15]: I do like the idea of having an apparent log out confirmation button, but I believe that consistency throughout the UI is best for the user (horizontal button layout similar to, if not exactly like, what you have for the `ConfirmationModal`)

Screen Shot 2021-09-20 at 14 22 59

**Review by jackthta — CHANGES_REQUESTED** [review]

File: src/components/interface/ConfirmationModal.vue (hunk 1 above)
Comment (SMITH1103) [reply src/components/interface/ConfirmationModal.vue:14]: Changed modal body text from "Enter password" to "Confirm your password" (the input placeholder text is part of a different component).

File: src/components/interface/ConfirmationModal.vue [hunk 3]
```diff
@@ -8,23 +8,23 @@
...
 
         <PasswordInput show-eye />
       </div>
 
-      <div class="flex flex-col items-center pb-4 mx-4">
+      <div class="pb-4 text-center">
         <Button
           color="green"
-          class="w-full p-4 mt-4"
+          class="w-full md:w-3/4 p-4 m-4 m-auto"
```
Comment (jackthta) [inline src/components/interface/ConfirmationModal.vue:20]: Alignment/spacing is a bit off before `md` breakpoint

Screen Shot 2021-09-22 at 13 10 36

File: src/components/interface/ConfirmationModal.vue [hunk 4]
```diff
@@ -54,5 +54,8 @@ export default defineComponent({
     title: { type: String, required: true },
   },
   emits: ["close"],
+  setup(){
+    console.log("from confimation modal");
```
Comment (jackthta) [inline src/components/interface/ConfirmationModal.vue:58]: Unsure if needed still :')

```suggestion
```

File: src/components/interface/LogoutModal.vue [hunk 5]
```diff
@@ -12,18 +12,18 @@
...
         <Button
           color="green"
-          class="w-full p-4 mt-4"
+          class="w-full md:w-3/4 p-4 m-4 m-auto"
```
Comment (jackthta) [inline src/components/interface/LogoutModal.vue:18]: Alignment/spacing is a bit off before `md` breakpoint

Screen Shot 2021-09-22 at 13 30 00

**Review by jackthta — CHANGES_REQUESTED** [review]

File: src/components/interface/ConfirmationModal.vue (hunk 3 above)
Comment (SMITH1103) [reply src/components/interface/ConfirmationModal.vue]: Adjusted margins to improve appearance on smaller screens.

File: src/components/interface/LogoutModal.vue (hunk 5 above)
Comment (SMITH1103) [reply src/components/interface/LogoutModal.vue:18]: Adjusted margin for smaller screens and to be consistent with confirmation modal

File: src/components/interface/ConfirmationModal.vue (hunk 4 above)
Comment (SMITH1103) [reply src/components/interface/ConfirmationModal.vue:58]: Removed

**Review by cacampbell — APPROVED** [review]

**Pull request merged**